src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -382,6 +385,23 @@ describe("Payment", () => {
     expect(component.find(ErrorModal).props().show).toBe(true)
   })
 
+  describe("Analytics", () => {
+    it("tracks click when use shipping address checkbox is unchecked", () => {
+      const { Component, dispatch } = mockTracking(PaymentRoute)
+      const component = mount(<Component {...testProps} />)
+      component
+        .find(Checkbox)
+        .at(0)
+        .simulate("click")
+      expect(dispatch).toBeCalledWith({
+        action_type: "Click",
+        subject: "use shipping address",
+        flow: "buy now",
+        type: "checkbox",
+      })
+    })
+  })
```

Comment:
Would be good to have a test for the case where no tracking should apply.

Reply:
Done!